content/download/osx.adoc

```diff
@@ -12,14 +12,14 @@ KiCad is supported on OS X 10.7 through OS X 10.12.
 
 - *KiCad extras* contains the footprints and all the instructions you need for making printed circuit board layouts while offline.
 
-=== Stable Release
+== Stable Release
```

Comment:
Why is this header level changed? It looks like we use three for all the other pages.